### PR TITLE
[chore] remove old codeowners

### PR DESCRIPTION
**Pull request by atoulme**
Remove previous codeowners, with their approval, in favor of new codeowners active on the repository.

@mstumpfx, @owais, @jrcamp, and @keitwb : thank you for your help!

### CODE REVIEW

File: .github/CODEOWNERS
Comment (MovieStoreGuy) [inline .github/CODEOWNERS:100]: Might be worth adding @jamesmoessis here as well as the other code owner of the docker receiver?

Comment (atoulme) [reply .github/CODEOWNERS:100]: let's do it :)

Comment (jamesmoessis) [reply .github/CODEOWNERS]: Happy to :)

**Review by mx-psi — APPROVED** [review]
LGTM, let's wait a few days before merging to give some time to former approvers to see this in case they still want to own these components

**Review by mstumpfx — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Review by dmitryax — APPROVED** [review]

**Comment by MovieStoreGuy** [comment]
@atoulme,

Is there a process you followed to help identify inactive code owners? Perhaps providing a means of generating the names to review?

I don't want to go fully automated and kick people, that feels rather cold but at least a documented way that we identify users would ensure there is no personal bias.

**Pull request merged**
